esp/esp/program/modules/handlers/resourcemodule.py

```diff
@@ -105,7 +105,7 @@ def resources_timeslot(self, request, tl, one, two, module, extra, prog):
 
             elif data['command'] == 'addedit':
                 #   add/edit timeslot
-                form = TimeslotForm(data, auto_id="timeslot_%s")
+                form = TimeslotForm(data, auto_id="timeslot_%s", program = prog)
```

Comment:
I now get an error when I try to add or edit a timeslot:

![image](https://user-images.githubusercontent.com/25392075/228399964-881dea31-ceca-4d45-a946-c9597a56fcf4.png)

![image](https://user-images.githubusercontent.com/25392075/228400181-cba0fd21-e6ef-4e97-8640-d487f76b96d8.png)